src/ldp/http/BasicResponseWriter.ts

```diff
@@ -34,6 +34,10 @@ export class BasicResponseWriter extends ResponseWriter {
 
     if (input.result.data) {
       input.result.data.pipe(input.response);
+      input.result.data.on('error', (error): void => {
+        this.logger.error(`Writing to HttpResponse failed with message ${error.message}`);
+        input.response.destroy(error);
```

Comment:
Seeing this here, why not use `pipeStreamsAndErrors`?

Reply:
Did it like this because I had to add the logger line anyway, but probably makes more sense to use the function and attach an error listener for the logging.